src/backend/web/static/swagger/api_v3.json

```diff
@@ -7445,6 +7529,11 @@
           },
           "backup": {
             "type": "object",
+            "nullable": true,
```

Comment:
is `backup` really both nullable and not required?

Reply:
Yeah.

Undefined in offseason: https://www.thebluealliance.com/api/v3/event/2022bc/alliances
Undefined in season: https://www.thebluealliance.com/api/v3/event/2022mabos/alliances

Nullable in season: https://www.thebluealliance.com/api/v3/event/2018nyut/alliances

Seems like if an event has a backup on any alliance, every other alliance object has a backup field present, which may be null or an actual backup

Reply:
amazing